src/main/scala/com/amazon/deequ/analyzers/jdbc/JdbcAnalyzer.scala

```diff
@@ -193,4 +193,13 @@ object Preconditions {
         s"but found ${JDBCType.valueOf(columnDataType).getName} instead!")
     }
   }
+
+  /** Statement has no ; outside of quotation marks (SQL injections)*/
+  def hasNoInjection(statement: Option[String]): Table => Unit = { _ =>
+    val pattern = """("[^"]*"|'[^']*'|[^;'"]*)*"""
+    val st = statement.get
+    if(!st.matches(pattern)){
```

Comment:
put a space betwenn ) and {
```suggestion
    if(!st.matches(pattern)) {
```